cmd/tester/main.go

```diff
@@ -16,16 +18,26 @@ func main() {
 		SegmentWriteKey     string `conf:"segment-write-key"        help:"writekey for the Segment project to send data to" validate:"nonzero"`
 		WebhookBucket       string `conf:"webhook-bucket"           help:"webhook bucket the Segment project sends data to" validate:"nonzero"`
 		WebhookAuthUsername string `conf:"webhook-auth-username"    help:"basic auth username for the webhook bucket the Segment project sends data to" validate:"nonzero"`
+		TestAllFixtures     bool   `conf:"test-all-fixtures"        help:"test all fixtures, regardless of individual failures"`
```

Comment:
I'm not 100% sure what the description of this flag means?  Could you clarify?

Reply:
the current version of `library-e2e-tester` terminates immediately the moment a fixture fails. This flag says "test all fixtures, even if some of them fail". I'm open to suggestions for a better name :)

Reply:
`IgnoreFailures`?